_data/navigation.yml

```diff
@@ -135,18 +134,21 @@ items:
               - url: /components/extractors/storage/google-drive/
                 title: Google Drive
 
-              - url: /components/extractors/storage/onedrive/
-                title: OneDrive
-
               - url: /components/extractors/storage/http/
                 title: HTTP
 
               - url: /components/extractors/storage/storage-api/
                 title: Keboola Connection Storage
 
+              - url: /components/extractors/storage/onedrive/
```

Comment:
nemohli bysme to dat podle abecedy?

Reply:
urcite mohli, mela jsem za to, ze jsem to vsude srovnala. A vono ne :-)).
Mas dobry oci! 
Dik

Reply:
aha, ja koukal na to jako na storage, uz to keboola uplne ignoruju :) 

oukej, nechme to takhle